data/quarg missions.txt

```diff
@@ -111,3 +111,75 @@ mission "First Contact: Kuwaru Efreti"
 				decline
+
+# The first time the player boards a disabled quarg ship, they will trigger this mission.
+# The Quarg give them the chance to remain on "friendly" terms if they do not take any technology.
+# If the player chooses to loot the ship, their Quarg reputation is set to -1000, and the 
+# condition "taken quarg technology" is set, which could be referenced in other missions, 
+# but currently causes the player to be followed and observed by occasional "Unknown Watcher" drones. 
+mission "First Quarg Boarding Action"
+	boarding
+	source 
+		government Quarg
+	on offer 
+		conversation 
+			`	You've been waiting for this moment for a long time. As far as you know, no human has ever disabled and boarded a Quarg ship. Even after the unfathomable amount of punishment that you unleashed upon it, you keep expecting the alien vessel to spring to life and either destroy you or vanish in a shimmer of blue light...but it seems even the Quarg have their limits. The ship is drifting, apparently without power.`
+			`	You lock your ship's boarding clamps into position and inspect the Quarg hull through your instruments, keeping one eye on your long-range scanners for any sign of more Quarg (though considering the way their ships can just materialize out of hyperspace, you don't expect a lot of warning). You identify the alien craft's armaments: fearsome twin beam weapons and powerful missile defense turrets. Of course they aren't fitted onto the standard turret-mounts you're used to seeing on human ships...so removing them without damage may be a challenge. And that's assuming no part of this alien ship explodes in your face.  It looks like the primary beams are at least 60 tons each...you have enough cargo space for those, right?`
+			`	A pit forms in your stomach...maybe this was a bad idea.` 
+			choice 
+				`	(There's no going back now. Keep scanning...look for the best way to slice open the hull.)`
+					goto hull
+				`	(Maybe it would be easier to capture the ship than to rip it apart. Try scanning the inside of the ship for defensive measures.)`
+					goto capture
+				`	(Abandon this insane project. No alien technology is worth your life.)`
+					# defer the mission to be tried again later and boarded ship is destroyed to prevent re-boarding.
+					goto selfdestruct 
+
+			label capture
+			`	You tune your sensors to inspect the living areas of the enemy ship, starting at what you're fairly certain is the airlock. It doesn't take long for your heart to sink at what you're seeing. There are literally hundreds of tiny, independent power sources distributed over the ships' internal surfaces. They appear to power some kind of intrusion countermeasures. Maybe with 1000 marines you would have a prayer of capturing this ship. Maybe.`
```

Comment:
Maybe you can replace 1000 with one thousand. 1000 just... looks a little out of place.